tests/integration/issues/2566_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package peer_test
```

Comment:
thought: unsure if this makes sense as the package name

Reply:
My bad, I copy-pasted from somewhere else and forgot to change this, renaming

- [ ] Rename package